data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+			`	You thank him for his help, and make your way back to the spaceport, mentally preparing yourself for what is likely to be a long meeting.`
+				accept
+
+	on complete
+		conversation
+			`Arriving on Hai-home, you manage to contact the Ministry of Culture, expressing your interests in obtaining Sheragi documents and manuals. After some time on hold, you're appointed to meet them in person.`
+			`	You get ready, and make your way to the city hall, where meet with the Hai officials, accompanied by a few diplomats. You all sit down on a round table, and they begin inquiring about the dig site on Zug, curious on the details of preservation, and what artifacts were found. They seem excited about it, and you realize that despite having encountered the Sheragi millenia ago, the Hai never had the opportunity to study their cities up close, and all they know are from hostile encounters or diplomatic meetings. They refuse to sell you documents, as this is the kind of knowledge money can't buy, but they suggest a cooperation between species. Given this opportunity, you tell them about the Sheragi black box found in the dig site, and the possible opportunity for decrypting it and uncovering it's secrets.`
+			`	"With the proper tools, we could decode it. But in return, we'd like to have a copy of the contents as well."`
+			`	You ponder your choices for a moment. There is no way you could solve this puzzle without their help. You've went this far, so you agree.`
+			`	"Excellent!"`
+			`	They chat with you a bit more, and the leader excuses himself for a moment to make a call. A moment later he returns.`
+			`	"We have arranged for a very capable team of archaeologists to meet you in the spaceport in a few hours. They specialize in Sheragi artifacts. I advise you to make room for 3 passengers and some cargo for their equipment."`
+			`	You thank them for their help, and leave the building, ready to meet your new team.`
+
+mission "The box 8"
+	description "Take the Hai archaeologists to meet Albert Foster on Zug."
+	name "The box"
+	source "Hai-home"
+	destination "Zug"
+	passengers 3
+	cargo "Archaeological Equipment" 1
+	
+	to offer
+		has "The box 7b: done"
+	
+	blocked `You arrive at the spaceport, but then you realize your ships don't have enough free bunks or cargo space required to bring the archaeologist team with you. You manage to contact them, and in an embarrassing conversation ask them to wait for you while you make sure you're ready.`
+	
+	on offer
+		conversation
+			`You meet up with the Hai archaeologists that will accompany you to human space, an all female team. They introduce themselves to you as Eyama, Rakei, and Desyo.`
+			`	Eyama is a linguist, specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`
```

Comment:
```suggestion
			`	Eyama is a linguist specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`
```
I mentioned earlier how Foster could have made some mention of having translated some of the Sheragi language. Having a character that actually knows the language though could lead to some interesting banter. "Actually, this is all wrong."